pkg/apis/crd.nsx.vmware.com/v1alpha2/ippool_types.go

```diff
@@ -0,0 +1,82 @@
+/* Copyright © 2023 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0 */
+
+package v1alpha2
```

Comment:
we also don't need v1alpha2 ipool, please remove it.

Reply:
ditto